cli/src/opts/multi_wallet.rs

```diff
@@ -125,6 +126,15 @@ pub struct MultiWallet {
     )]
     pub mnemonic_paths: Option<Vec<String>>,
 
+    #[clap(
+        long = "mnemonic-derivation-paths",
+        alias = "hd-paths",
+        help_heading = "WALLET OPTIONS - RAW",
+        help = "The wallet derivation path. Works with both --mnemonic-path and hardware wallets.",
+        value_name = "PATHS"
+    )]
+    pub hd_paths: Option<Vec<String>>,
```

Comment:
this requires `mnemonic_paths`, right?

so we can restrict this with another clap attribute

` requires = "mnemonic-paths"`

Reply:
Not necessarily, it can be used with hardware wallets instead. What we can do is add `conflicts_with` to make `mnemonic-path` and `mnemonic-index` mutually exclusive to avoid confusion.

Reply:
sgtm!